src/main/resources/com/cloudbees/simplediskusage/QuickDiskUsagePlugin/refresh-disk-usage.js

```diff
@@ -0,0 +1,10 @@
+function refreshDiskUsage(a) {
+
+    fetch("refresh", {
+        method: "POST",
+        headers: crumb.wrap({})
+    }).catch(() => {});
```

Comment:
This is to avoid an error message in the console. If I exclude it, the console prints:
`Unhandled Promise Rejection: TypeError: Load failed`

Reply:
I believe the difference is that the old onClick approach immediately triggered a page refresh, so the unresolved fetch promise was interrupted before any rejection could be reported. The new approach using js callbacks keeps the promise alive in the current page context, so we need to handle the promise with .catch()

Reply:
Ignoring all errors indiscriminately smells wrong. See e.g. articles like https://stackify.com/best-practices-exceptions-java/. Errors should be exceptional, so either an error should not be generated in this case, or we should be catching a specific error (with a comment explaining why it is expected) rather than catching all errors indiscriminately without reasoning.

Reply:
@basil 

Looking into this further and discovered some broken functionality that I am investigating. Marking as draft in meantime.